docs/source/getting-started.rst

```diff
@@ -57,4 +66,19 @@ The two suites you will always be able to select between is a "simplified" comma
 DataLad extensions can add additional suites when you install them.
 Please note that we recommend the "simplified" command suite to users, as the complete suite can contain experimental implementations.
 
-.. image:: _static/suite_menu.png
\ No newline at end of file
+.. image:: _static/suite_menu.png
+
+The Utilities and Help Tab
```

Comment:
Here and below "tab" should be "menu", I think

Reply:
done